pkg/config/containers.conf

```diff
@@ -419,7 +419,7 @@ default_sysctls = [
 # Format is a single character [a-Z] or a comma separated sequence of
 # `ctrl-<value>`, where `<value>` is one of:
 # `a-z`, `@`, `^`, `[`, `\`, `]`, `^` or `_`
-#
+# Specifying “” disables this feature.
```

Comment:
```suggestion
# Specifying "" disables this feature.
```
Seems like you are not using the right quotes, just in case some tries to copy them we should fix it.